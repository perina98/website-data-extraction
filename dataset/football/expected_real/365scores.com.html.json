[
    {
        "team2": "Burnley",
        "time": "19:00",
        "team1": "Leicester"
    },
    {
        "team2": "Sheffield Utd",
        "time": "19:00",
        "team1": "Aston Villa"
    },
    {
        "team2": "Palace",
        "time": "21:15",
        "team1": "Man Utd"
    },
    {
        "team2": "Brest",
        "time": "19:00",
        "team1": "Dijon"
    },
    {
        "team2": "Lyon",
        "time": "19:00",
        "team1": "Rennes"
    },
    {
        "team2": "Metz",
        "time": "19:00",
        "team1": "Angers"
    },
    {
        "team2": "OGC Nice",
        "time": "19:00",
        "team1": "Nimes"
    },
    {
        "team2": "Saint-Ã‰tienne",
        "time": "19:00",
        "team1": "Lens"
    },
    {
        "team2": "Bordeaux",
        "time": "21:00",
        "team1": "PSG"
    },
    {
        "team2": "Lille",
        "time": "21:00",
        "team1": "Marseille"
    },
    {
        "team2": "Montpellier",
        "time": "21:00",
        "team1": "Lorient"
    },
    {
        "team2": "Strasbourg",
        "time": "21:00",
        "team1": "Monaco"
    },
    {
        "team2": "Nantes",
        "time": "21:00",
        "team1": "Reims"
    },
    {
        "team2": "Rot-Weiss Essen",
        "time": "18:30",
        "team1": "Holstein Kiel"
    },
    {
        "team2": "RB Leipzig",
        "time": "20:45",
        "team1": "Wolfsburg"
    },
    {
        "team2": "Gladbach",
        "time": "20:45",
        "team1": "Dortmund"
    },
    {
        "team2": "Juventus",
        "time": "20:45",
        "team1": "Spezia"
    },
    {
        "team2": "Liverpool",
        "time": "21:15",
        "team1": "Chelsea"
    }
]