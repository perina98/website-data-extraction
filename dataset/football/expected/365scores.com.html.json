[
    {
        "team1": "Burnley",
        "time": "19:00",
        "team2": "Leicester"
    },
    {
        "team1": "Sheffield Utd",
        "time": "19:00",
        "team2": "Aston Villa"
    },
    {
        "team1": "Palace",
        "time": "21:15",
        "team2": "Man Utd"
    },
    {
        "team1": "Brest",
        "time": "19:00",
        "team2": "Dijon"
    },
    {
        "team1": "Lyon",
        "time": "19:00",
        "team2": "Rennes"
    },
    {
        "team1": "Metz",
        "time": "19:00",
        "team2": "Angers"
    },
    {
        "team1": "OGC Nice",
        "time": "19:00",
        "team2": "Nimes"
    },
    {
        "team1": "Saint-Ã‰tienne",
        "time": "19:00",
        "team2": "Lens"
    },
    {
        "team1": "Bordeaux",
        "time": "21:00",
        "team2": "PSG"
    },
    {
        "team1": "Lille",
        "time": "21:00",
        "team2": "Marseille"
    },
    {
        "team1": "Montpellier",
        "time": "21:00",
        "team2": "Lorient"
    },
    {
        "team1": "Strasbourg",
        "time": "21:00",
        "team2": "Monaco"
    },
    {
        "team1": "Nantes",
        "time": "21:00",
        "team2": "Reims"
    },
    {
        "team1": "Rot-Weiss Essen",
        "time": "18:30",
        "team2": "Holstein Kiel"
    },
    {
        "team1": "RB Leipzig",
        "time": "20:45",
        "team2": "Wolfsburg"
    },
    {
        "team1": "Gladbach",
        "time": "20:45",
        "team2": "Dortmund"
    },
    {
        "team1": "Juventus",
        "time": "20:45",
        "team2": "Spezia"
    },
    {
        "team1": "Liverpool",
        "time": "21:15",
        "team2": "Chelsea"
    }
]